src/confcom/azext_confcom/tests/latest/test_confcom_virtual_node.py

```diff
@@ -53,6 +59,44 @@ class PolicyGeneratingVirtualNode(unittest.TestCase):
             ]
         }
         """
+    custom_json2 = """
+{
+  "version": "1.0",
+  "fragments": [],
```

Comment:
Do we have any examples in the confcom dir that need to be updated?

Reply:
I need to add docs about it. Will do that now